### ISSUE TITLE
Agent prompt to answer question is not accessible

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#291971]
When focus is in the accessible view, a user is unaware that a prompt has appeared that they needc to answer. We should alert, play the user confirmation required signal, and provide a command and keybinding to focus the element, including this in the accessibility help dialog.

The selector: `div.chat-question-carousel-container`
